apis/beacon/blocks/root.yaml

```diff
@@ -30,9 +30,8 @@ get:
                 type: object
                 properties:
                   root:
-                    allOf:
-                      - description: HashTreeRoot of BeaconBlock/BeaconBlockHeader object
-                      - $ref: '../../../beacon-node-oapi.yaml#/components/schemas/Root'
+                    $ref: '../../../beacon-node-oapi.yaml#/components/schemas/Root'
+                    description: HashTreeRoot of BeaconBlock/BeaconBlockHeader object
```

Comment:
This is not working in swagger

Reply:
How are you testing swagger exactly? What does it mean "not working"? Please share reproduction steps and I can look for alternative solutions

Reply:
Are you running bundle before serving the file?

```
swagger-cli bundle ./beacon-node-oapi.yaml -r -t yaml -o ./deploy/beacon-node-oapi.yaml
```

 I get different results than you

Reply:
@dapplion You probably didn't switch schema to "dev" default it latest version on github without your changes

Reply:
Sorry I still follow how are you getting a `$ref` property rendered in swagger. The pre-processing `swagger-cli bundle` resolves all $ref such that the resulting yaml has cleanly merge properties. So to clarify:

(a)
- Serve root `./beacon-node-oapi.yaml` and render that in swagger

(b)
- You run `swagger-cli bundle ./beacon-node-oapi.yaml -r -t yaml -o ./deploy/beacon-node-oapi.yaml`
- Serve `./deploy/beacon-node-oapi.yaml` and render that in swagger